tests/cross_functional/system_test/test_mcg_noobaa_db_ha_and_feature_verification.py

```
@@ -0,0 +1,960 @@
+            sleep=VERSION_EXPIRATION_SLEEP,
+            func=get_obj_versions,
+            mcg_obj=mcg_obj,
+            awscli_pod=awscli_pod_session,
+            bucket_name=versioning_bucket_post_rebuild.name,
+            obj_key=versioning_key,
+        ):
+            if len(versions) == 1:
+                logger.info("Non-current versions expired; current version retained")
+                break
+        else:
+            assert False, "Non-current versions were not expired in time"
```

Ruff (0.15.17)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)